src/picongpu/include/plugins/particleMerging/VoronoiCell.hpp

```diff
@@ -0,0 +1,261 @@
+/* Copyright 2017 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "algorithms/KinEnergy.hpp"
+#include "pmacc_types.hpp"
+
+namespace picongpu
+{
+namespace plugins
+{
+namespace particleMerging
+{
+
+    /* Status of a Voronoi cell
+     *
+     * collecting:
```

Comment:
Please check [this](http://stackoverflow.com/a/13756348) short HOWTO document enums with doxygen